docs/swagger_v2.yml

```diff
@@ -145,6 +145,110 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /apps/{app}/calls/{call}/log:
+    get:
+      summary: Get call logs
+      description: Get call logs
+      tags:
+        - Call
+        - Log
+      parameters:
+        - name: app
+          description: App Name
+          required: true
+          type: string
+          in: path
+        - name: call
+          description: Call ID.
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
+          description: Log found
+          schema:
+            $ref:  '#/definitions/LogWrapper'
+        404:
+          description: Log not found.
+          schema:
+            $ref: '#/definitions/Error'
+
+  /apps/{app}/calls/{call}:
+    get:
+      summary: Get call information
+      description: Get call information
+      tags:
+        - Call
+      parameters:
+        - name: app
+          description: app name
+          required: true
+          type: string
+          in: path
+        - name: call
+          description: Call ID.
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
+          description: Call found
+          schema:
+            $ref:  '#/definitions/CallWrapper'
+        404:
+          description: Call not found.
+          schema:
+            $ref: '#/definitions/Error'
+
+  /apps/{app}/calls:
```

Comment:
In v2 we had a flat API hierarchy.
To align with this we'd have `/calls` with a mandatory query parameter of `app` rather than `/apps/{app}/calls`